cmd/minishift/cmd/start.go

```diff
@@ -170,7 +170,10 @@ func runStart(cmd *cobra.Command, args []string) {
 	proxyConfig := handleProxies()
 
 	// preflight check (before start)
-	preflightChecksBeforeStartingHost()
+	// If --skip-preflight-check flag is set to true means do not check
+	if !viper.GetBool(configCmd.SkipCheckPreflight.Name) {
+		preflightChecksBeforeStartingHost()
```

Comment:
can we move this instead to the `preflight.go` file? and use it to bailout on the entrance of the function `preflightChecksBeforeStartingHost()` ?

Reply:
I do not have strong opinion on this.